packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/IntFixedEnum.Serialization.cs

```diff
@@ -0,0 +1,29 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace UnbrandedTypeSpec.Models
+{
+    internal static partial class IntFixedEnumExtensions
```

Comment:
Lets name the class and file the same.

Reply:
Emmmm do we want to change this here? We have the same implementation in every SDK now.
I think it is better that we open an issue to fix it here and in autorest.csharp if we decide to do this

Reply:
this file is not `serialization` its just some extension methods and the filename should match the class.

Reply:
I prefer we first get everything ready and identical (in generated code) with the things we have `autorest.csharp`.
And we fix this in the next PR on both sides.

Reply:
Fixed.
I will open another PR in autorest.csharp to align this change.